tests/endpoints/test_operations_log.py

```diff
@@ -42,13 +42,14 @@ def test_get_static_collection(self):
                 'ticket_slug': str(uuid.uuid4()),
                 'version': str(uuid.uuid4()),
             }
-            records.append(record)
+            # records.append(record)
```

Comment:
I think we can just remove these comments?

Reply:
Force pushed over the last commit to clean these up.  Thanks for the catch.